catalogs/de.cat

```diff
@@ -21,7 +21,7 @@
 #Datei: dimension.xaf
 
 fmath "Die Mathematik hinter Fraktalen"
-fmath1 "Fraktale Geometrie ist ein sehr junges
+fmath1 "Fraktale Geometry ist ein sehr junges
```

Comment:
German "Geometrie" is correct.
Did you run "codespell" over foreign languages other than English? This won't work, most probably.